parlai/utils/data.py

```diff
@@ -140,9 +135,30 @@ def split_domains_by_fold(
             start = train_frac + valid_frac
             end = 1.0
 
+        random.Random(seed).shuffle(data)
+        return data[int(start * len(data)) : int(end * len(data))]
+
+    @classmethod
+    def split_domains_by_fold(
```

Comment:
maybe shards? Or subsets? This is fine but `domains` is somewhat a specific name relative to how this function could be used?

Reply:
Changed to "subsets"